probe-rs-tools/src/bin/probe-rs/cmd/complete.rs

```diff
@@ -255,15 +255,16 @@ impl ShellExt for Zsh {
 
 impl ShellExt for Bash {
     fn install(&self, file_name: &str, script: &str) -> Result<()> {
+        const DEST: &str = ".local/share/bash-completion/completions/";
```

Comment:
I am not sure this is the correct place. Source: https://serverfault.com/questions/506612/standard-place-for-user-defined-bash-completion-d-scripts

Reply:
It is, according to the FAQs (https://github.com/scop/bash-completion):

> Q. Where should I install my own local completions?
> 
> A. Put them in the completions subdir of $BASH_COMPLETION_USER_DIR (defaults to $XDG_DATA_HOME/bash-completion or ~/.local/share/bash-completion if $XDG_DATA_HOME is not set) to have them loaded automatically on demand when the respective command is being completed.

Reply:
The FAQ states `Alternatively, you can write them directly in ~/.bash_completion which is loaded eagerly by our main script.` just a line below. That file can always be created, whereas `.local/share` does for sure not exist on macOS and I am skeptical whether `bash_completion` regards it on macOS. `$XDG_DATA_HOME` for sure does not exist. That's a Linux thing. So we need to make sure it works for both.

Reply:
Bash is not the default shell on macOS, and it also doesn't seem to have the bas_completion installed by default. So I don't think we should put too much weight on that use case.

If `~/.bash_completion` already exists, we currently just seem to overwrite it with our completions. Using a separate file seems better to me.